docs/core/project-sdk/msbuild-props.md

```diff
@@ -699,6 +600,116 @@ The `EnableDynamicLoading` property indicates that an assembly is a dynamically
 </PropertyGroup>
 ```
 
+## Items
+
+[MSBuild items](/visualstudio/msbuild/msbuild-items) are inputs into the build system. Items are specified according to their type, which is the element name. For example, `Compile` and `Reference` are two [common item types](/visualstudio/msbuild/common-msbuild-project-items). The following additional item types are made available by the .NET SDK:
+
+- [PackageReference](#packagereference)
+- [ProjectReference](#projectreference)
+- [Reference](#reference)
+- [TrimmerRootAssembly](#trimmerrootassembly)
+
+You can use any of the standard [item attributes](/visualstudio/msbuild/item-element-msbuild#attributes-and-elements), for example, `Include`, `Version`, and `Update`, on these items.
+
+### PackageReference
+
+The `PackageReference` item defines a reference to a NuGet package.
+
+The `Include` attribute specifies the package ID. The `Version` attribute specifies the version or version range. For information about how to specify a minimum version, maximum version, range, or exact match, see [Version ranges](/nuget/concepts/package-versioning#version-ranges).
```

Comment:
The issue this PR fixes asks about a PackageReference with Update attribute -- this doesn't answer the question why it would be / why you would use Update instead of Include.

Reply:
Daniel suggested adding a note at the top of the article (I ended up putting it at the top of the Items section) that any of the standard item attributes can be used (like Update), and link to the MSBuild article about it. I did that [here](https://review.docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props?branch=pr-en-us-23373#items).

Reply:
That only states the attributes are available, it doesn't explain why you would use Update despite the fact that throughout this article Include is used.

Reply:
`Include` adds a new item, `Update` modifies an existing one.  `Update` would most often be used to modify an item that is already implicitly added by the .NET SDK.

Reply:
But why would it be used on a package reference instead of Include?

Reply:
I think you get a build error if you try to include an item twice.

Reply:
Prior to .NET Core 3.0, there were some `PackageReference` items that are implicitly added by the .NET SDK.  (In .NET Core 3.0 and higher, there are implicit `FrameworkReference` items instead).  If you try to add a `PackageReference` that is already included by default, you'll get the following warning:

> NETSDK1023: A PackageReference for '{0}' was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs